### PR TITLE
Start game

### PR DESCRIPTION
**Pull request by paula-roesler**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,49 +1,46 @@
-// import PropTypes from 'prop-types'
```
Comment (AnastasiyaVasileuskaya) [inline src/App.js:1]: You can delete this comment before you merge

File: src/App.spec.js [hunk 2]
```diff
@@ -1,17 +1,41 @@
 import { render, screen } from '@testing-library/react'
+import { BrowserRouter as Router } from 'react-router-dom'
+
+// import userEvent from '@testing-library/user-event'
+// import { createMemoryHistory } from 'history'
```
Comment (AnastasiyaVasileuskaya) [inline src/App.spec.js:5]: You can delete this comment before you merge

File: src/App.spec.js [hunk 3]
```diff
@@ -1,17 +1,41 @@
...
+import React from 'react'
+// import { Router } from 'react-router-dom'
```
Comment (AnastasiyaVasileuskaya) [inline src/App.spec.js:7]: the same

File: src/App.spec.js [hunk 4]
```diff
@@ -1,17 +1,41 @@
...
 
+// it('renders app title', () => {
+//   render(
+//     <Router>
+//       <App />
+//     </Router>
+//   )
+//   expect(screen.getByText('GolfGamer')).toBeInTheDocument()
+// })
+
```
Comment (AnastasiyaVasileuskaya) [inline src/App.spec.js:19]: the same

File: src/components/Button/Button.js [hunk 5]
```diff
@@ -2,15 +2,35 @@ import styled from 'styled-components/macro'
...
+  text-transform: uppercase;
+  letter-spacing: 0.3em;
   border: none;
+  border-radius: 4px;
   padding: 10px;
   width: 100%;
+  height: 40px;
+  color: ${props => (props.txtcolor ? props.txtcolor : 'var(--white)')};
+  background-color: ${props =>
+    props.bgcolor ? props.bgcolor : 'var(--primary)'};
```
Comment (AnastasiyaVasileuskaya) [inline src/components/Button/Button.js:18]: Cool!Conditional styling!

File: src/components/Button/Button.js [hunk 6]
```diff
@@ -2,15 +2,35 @@ import styled from 'styled-components/macro'
...
+//   &:focus,
+//   &:hover,
+//   &:visited,
+//   &:link,
+//   &:active {
+//     text-decoration: none;
+//   }
+// `
+
+// export default props => <StyledLink {...props} />
```
Comment (AnastasiyaVasileuskaya) [inline src/components/Button/Button.js:36]: You can delete this comment before you merge

File: src/components/GamePage/GamePage.spec.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import { screen, render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import { act } from 'react-dom/test-utils'
+
+import GamePage from './GamePage'
+// import userEvent from '@testing-library/user-event'
+// import Button from '../Button'
```
Comment (AnastasiyaVasileuskaya) [inline src/components/GamePage/GamePage.spec.js:7]: You can delete this comment before you merge

File: src/components/GamePage/__snapshots__/GamePage.spec.js.snap [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+      </button>
+    </div>
+  </section>
+  <a
+    class="Button-sc-19qr0ud-0 hgHwpI"
+    href="/"
+  >
+    End game
+  </a>
+</div>
```
Comment (AnastasiyaVasileuskaya) [inline src/components/GamePage/__snapshots__/GamePage.spec.js.snap:52]: It's great that you use snapshot test!

File: src/components/NewGamePage/NewGamePage.js [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+import Button from '../Button'
+import { NavLink } from 'react-router-dom'
+
+export default function NewGamePage({ addPlayer, players, resetForm, name }) {
+  return (
+    <>
+      <AddPlayerForm
+        key={name}
+        role="form"
+        // name="addPlayerForm"
```
Comment (AnastasiyaVasileuskaya) [inline src/components/NewGamePage/NewGamePage.js:12]: You can delete this comment before you merge.

File: src/components/NewGamePage/NewGamePage.spec.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+    expect(screen.getByRole('textbox')).toHaveValue('Sue')
+  })
+
+  //   it('applies the name of the player to the submit callback', () => {
+  //     const callback = jest.fn()
+  //     render(<AddPlayerForm onAddPLayer={callback} />)
+  //     userEvent.type(screen.getByLabelText('Add player:'), 'Sue')
+  //     userEvent.click(screen.getByRole('button'))
+  //     expect(callback).toHaveBeenCalledWith({ nameOfPlayer: 'Sue' })
+  //   })
```
Comment (AnastasiyaVasileuskaya) [inline src/components/NewGamePage/NewGamePage.spec.js:33]: You can delete this comment before you merge.

File: src/components/GlobalStyle.js [hunk 11]
```diff
@@ -1,6 +1,14 @@
 import { createGlobalStyle } from 'styled-components'
 
 export default createGlobalStyle`
+  :root {
+    --white: #fcfcfc;
+    --background: #000;
+    --primary: royalblue;
+    --secondary: lightgreen;
+    --disabled: #ABBDCF;
+  }
```
Comment (AnastasiyaVasileuskaya) [inline src/components/GlobalStyle.js:10]: I like that you use variables in global style!

**Review by AnastasiyaVasileuskaya — COMMENTED** [review]
Very good!Keep-up the great work!

File: src/components/NewGamePage/NewGamePage.js [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+      <AddPlayerForm
+        key={name}
+        role="form"
+        onAddPlayer={addPlayer}
+        disabled={players.length >= 4}
+        placeholderText={
+          players.length >= 4
+            ? 'Your flight has reached the max. of 4 players'
+            : 'Enter player name (e.g. Joe)'
+        }
```
Comment (aseeba) [inline src/components/NewGamePage/NewGamePage.js:18]: Great idea to limit the number of players and set the input field to "disabled" so it fits. Good work! <3

**Review by aseeba — COMMENTED** [review]
Great work! 🥇 
You can be proud of yourself

**Pull request merged**